src/V/index.mjs

```diff
@@ -1378,6 +1381,83 @@ const V = (function() {
         return xml;
     };
 
+    const _attributeNames = Object.create(null);
+
+    // List of attributes for which not to split camel case words.
+    [
+        'baseFrequency',
+        'baseProfile',
+        'clipPathUnits',
+        'contentScriptType',
+        'contentStyleType',
+        'diffuseConstant',
+        'edgeMode',
+        'externalResourcesRequired',
+        'filterRes', // deprecated
+        'filterUnits',
+        'gradientTransform',
+        'gradientUnits',
+        'kernelMatrix',
+        'kernelUnitLength',
+        'keyPoints',
+        'lengthAdjust',
+        'limitingConeAngle',
+        'markerHeight',
+        'markerUnits',
+        'markerWidth',
+        'maskContentUnits',
+        'maskUnits',
+        'numOctaves',
+        'pathLength',
+        'patternContentUnits',
+        'patternTransform',
+        'patternUnits',
+        'pointsAtX',
+        'pointsAtY',
+        'pointsAtZ',
+        'preserveAlpha',
+        'preserveAspectRatio',
+        'primitiveUnits',
+        'refX',
+        'refY',
+        'requiredExtensions',
+        'requiredFeatures',
+        'specularConstant',
+        'specularExponent',
+        'spreadMethod',
+        'startOffset',
+        'stdDeviation',
+        'stitchTiles',
+        'surfaceScale',
+        'systemLanguage',
+        'tableValues',
+        'targetX',
+        'targetY',
+        'textLength',
+        'viewBox',
+        'viewTarget', // deprecated
+        'xChannelSelector',
+        'yChannelSelector',
+        'zoomAndPan' // deprecated
+    ].forEach((name) => _attributeNames[name] = name);
+
+    const attributeNames = new Proxy(_attributeNames, {
+        get(cache, name) {
```

Comment:
The `cache` here is the `_attributeNames` object itself, right? I think we should add comments about what is happening in this code because it is very complex... this is an example of a memoized getter, right? (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#smart_self-overwriting_lazy_getters)